cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/SingleBrokerStats.java

```diff
@@ -53,8 +53,8 @@ public boolean isEstimated() {
    * Return an object that can be further used
    * to encode into JSON
    */
-  public Map<String, Object> getJSONStructure() {
-    Map<String, Object> entry = _basicStats.getJSONStructure();
+  public Map<String, Object> getJsonStructure() {
```

Comment:
now we have `SingleBrokerCapacityStats`, looks like this can be renamed as `SingleBrokerUtilizationStats`